demos/http/http_demo_s3_generate_presigned_url/http_demo_s3_generate_presigned_url.c

```diff
@@ -226,7 +226,8 @@ struct NetworkContext
  */
 static bool printS3ObjectFilePresignedURL( const char * pHost,
                                            size_t hostLen,
-                                           const char * pPath );
+                                           const char * pPath,
+                                           const bool is_get );
```

Comment:
Can we please change the variable name to `isGet` to keep the naming consistent?

Reply:
certainly